api.go

```diff
@@ -81,7 +81,7 @@ func (server *ApiServer) Listen(host string, port string) {
 		"host":    host,
 		"port":    port,
 		"version": Version,
-	}).Info("API HTTP server starting")
+	}).Printf("Toxiproxy HTTP server starting on endpoint %s:%s", host, port)
```

Comment:
It could be used [`Infof`](https://github.com/sirupsen/logrus/blob/79c5ab66aa2ce7d9ff7b3c437ebc22fcc519a967/entry.go#L361), so it would be more explicit which level is used.

Also I try avoid using the application name in the logs like `Toxiproxy`. It is already in the name of the binary `toxiproxy-server`. Otherwise it would overloaded with Toxiproxy word in logs, that most of time ignored.